src/track/taglib/trackmetadata_id3v2.cpp

```diff
@@ -382,10 +382,10 @@ void writeTextIdentificationFrame(
         auto pFrame =
                 std::make_unique<TagLib::ID3v2::TextIdentificationFrame>(id, stringType);
         pFrame->setText(toTString(text));
-        pTag->addFrame(pFrame.get());
-        // Now that the plain pointer in pFrame is owned and managed by
-        // pTag we need to release the ownership to avoid double deletion!
-        pFrame.release();
+
+        // Release returns a pointer to the managed object and releases the ownership,
```

Comment:
The comment is obsolete after dropping the 2-step get-release pattern. Repeating the std definition  of release() is pointless. Also applies to other locations in the code.

The 2-step pattern is required for exception safe C++ programming. But since any exception thrown by addFrame() is expected to be fatal a resource leak doesn't matter then.

Reply:
This code has been written with purpose and the comment explained why. If you decide to take a simpler approach then delete the obsolete comment. I don't care, does not really matter. But please change it in a consistent way.

Reply:
The resource leak reported by Coverity is obviously a false positive.